components/raftstore/src/store/worker/split_config.rs

```diff
@@ -6,12 +6,16 @@ use lazy_static::lazy_static;
 use online_config::{ConfigChange, ConfigManager, OnlineConfig};
 use parking_lot::Mutex;
 use serde::{Deserialize, Serialize};
-use tikv_util::{config::VersionTrack, info};
+use tikv_util::{
+    config::{ReadableSize, VersionTrack},
+    info,
+};
 
 const DEFAULT_DETECT_TIMES: u64 = 10;
 const DEFAULT_SAMPLE_THRESHOLD: u64 = 100;
 pub(crate) const DEFAULT_SAMPLE_NUM: usize = 20;
-const DEFAULT_QPS_THRESHOLD: usize = 3000;
+pub const DEFAULT_QPS_THRESHOLD: usize = 3000;
+pub const DEFAULT_BIG_REGION_QPS_THRESHOLD: usize = 7000;
 const DEFAULT_BYTE_THRESHOLD: usize = 30 * 1024 * 1024;
```

Comment:
we should increase the bytes-threshhold as well, for example, in the stability test, we disabled load based split by increase the byte-threshhold

`SET config tikv split.byte-threshold=1572864000;`